src/components/common/YourCompany/EntityName.vue

```diff
@@ -137,8 +159,8 @@
               <span>{{ getEditLabel }}</span>
             </v-btn>
             <span
-              v-if="hasCompanyNameChanged || (hasBusinessNameChanged &&
-                (isAlterationFiling || isFirmChangeFiling || isSpecialResolutionFiling))"
+              v-if="(hasCompanyNameChanged || (hasBusinessNameChanged &&
```

Comment:
Already too long, should be converted into computed

Reply:
When you change this to a computed (get) then you also have the opportunity to add a comment to describe the particular business logic (which I recommend doing, if easy).